src/clr-core/tag/tag.element.ts

```diff
@@ -40,16 +38,24 @@ export class CwcTag extends CwcBaseButton {
   /** Sets the color of the tag (and badge if present) from a predefined list of choices */
   @property({ type: String })
   color: 'gray' | 'purple' | 'blue' | 'orange' | 'light-blue';
-
   static get styles() {
     return [baseStyles, styles];
   }
+
+  render() {
+    return html`
+    <div class="private-host">
+      <slot></slot>
+    </div>
+    ${this.hiddenButtonTemplate}
```

Comment:
Does `hiddenButtonTemplate` always need to be here? My question/concern is that this item captures focus even when it's not clickable.

Reply:
Hmm I'll double check this. Every button that extends the base button has this. It should definitely not capture focus ever and should be completely hidden. It's only used as a workaround for triggering form submit events.

Reply:
I double checked and this works as expected and does not capture focus. If the button is set to readonly it completely removes it from the DOM.